modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl

```diff
@@ -2,15 +2,16 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  *}
+{$componentName = 'search-filters-modules'}
 
 <div id="search_filters_brands">
-  <section class="facet">
+  <section class="facet {$componentName}">
```

Comment:
```suggestion
  <section class="{$componentName} facet">
```

Nitpick but everywhere we start by our personal classes then add every other classes (externals, bootstraps..)